database/index.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 const Review = require('./models/reviews');
-const reviews = require('./data.js');
-const moment = require('moment');
+// const moment = require('moment');
+// const reviews = require('./data.js');
```

Comment:
Did you keep these here and commented on purpose? Could clean up a little if you don't need it after all!

Reply:
It was just for seeding the db, I am going to keep them for now just in case I have to reseed again for some reason